doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_reset.rst

```diff
@@ -0,0 +1,62 @@
+.. _ug_nrf54h20_architecture_reset:
+
+nRF54H20 reset behavior
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. TODO
+
+nrfutil allows for different types of reset behavior on the various cores of the nRF54H20 SoC, based on the current lifecycle state of the device.
```

Comment:
nRF Util behavior should be documented in the nRF Util docs, not in the NCS docs. Especially when we discuss the behavior with a specific SoC, not related directly to the NCS.